express-get-json/server.js

```diff
@@ -0,0 +1,49 @@
+import express from 'express';
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95,
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100,
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92,
+  },
+};
+
+const app = express();
+
+app.use(function (req, res, next) {
+  console.log('Time: ', Date.now());
+  console.log('method: ', req.method);
+  next();
+});
+
+app.get('/api/grades', (req, res) => {
+  const gradeArray = [];
+
+  for (const id in grades) {
+    const grade = grades[id];
+    gradeArray.push({
```

Comment:
As you see, you are just pulling in the fields from `grade` and copying them into a new object instance. It would work just as well to just use `gradeArray.push(grade)`.